src/components/FriendsPage/FriendsPage.tsx

```diff
@@ -0,0 +1,93 @@
+import React, { useState } from 'react';
+import ProfileList from './ProfileList';
+import { useRecoilValue } from 'recoil';
+import { friendsInfoSelector } from '../../state/atom';
+import Profile from './Profile';
+import styled from 'styled-components';
+import { AiOutlineSearch } from 'react-icons/ai';
+
+const FriendsPage = () => {
+  const [isSearching, setIsSearching] = useState(false);
+  const [text, setText] = useState('');
+  const friendsInfo = useRecoilValue(friendsInfoSelector);
+
+  const handleChange = (e: React.ChangeEvent<HTMLInputElement>) => {
+    setText(e.target.value);
+  };
+
+  return (
+    <FriendsPageBox>
+      <Header>
+        <h1>친구</h1>
+        <button onClick={() => setIsSearching(!isSearching)}>
+          <AiOutlineSearch size={30} />
+        </button>
+      </Header>
+
+      {isSearching ? (
+        <>
+          <InputBox>
+            <SearchInput
+              placeholder="친구 검색"
+              value={text}
+              onChange={handleChange}
+            />
+          </InputBox>
+          <>
+            {friendsInfo.map((friend) => (
+              <>
+                {friend.userName.includes(text) ? (
+                  <Profile
+                    key={friend.userId}
+                    userName={friend.userName}
+                    userImg={friend.userImg}
+                    statusMessage={friend.statusMessage}
+                  />
+                ) : (
+                  <></>
+                )}
+              </>
+            ))}
```

Comment:
이부분은 저는 이렇게 작성했습니다. 함수를 두개 겹쳐서 사용해도 괜찮은 것 같아요
```suggestion
            {friendsInfo.filter((itmes) => items.userName.inclues(text)).map((friend) => {
            <Profile
                    key={friend.userId}
                    userName={friend.userName}
                    userImg={friend.userImg}
                    statusMessage={friend.statusMessage}
                    />
                    })}
```
괄호가 쌍이 맞나 모르겠네요 ㅎㅎ